Tests/Runner/BranchedWorkspaceTest.php

```diff
@@ -0,0 +1,281 @@
+<?php
+
+namespace Keboola\DockerBundle\Tests\Runner;
+
+use Keboola\Csv\CsvFile;
+use Keboola\DockerBundle\Docker\Component;
+use Keboola\DockerBundle\Docker\JobDefinition;
+use Keboola\DockerBundle\Docker\Runner;
+use Keboola\DockerBundle\Docker\Runner\UsageFile\NullUsageFile;
+use Keboola\DockerBundle\Tests\BaseRunnerTest;
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApi\Components;
+use Keboola\StorageApi\DevBranches;
+use Keboola\StorageApi\Options\Components\Configuration;
+use Keboola\StorageApiBranch\ClientWrapper;
+use Keboola\Temp\Temp;
+
+class BranchedWorkspaceTest extends BaseRunnerTest
+{
+    const BRANCH_NAME = 'workspace-test';
+    const BUCKET_NAME = 'workspace-test';
+    const TABLE_NAME = 'test-table';
+
+    /**
+     * @doesNotPerformAssertion
+     */
```

Comment:
to by ale melo udelat assert - ze opravdu doslo k te kopii

Reply:
Nestaci se spolehnout na to, ze run job nefailnul?

Reply:
jako pro účely toho, co jsme tady měnili jo. Je to takovej "regression test". Na druhou stranu "je škoda" si neověřit, že se ten výsledek opravdu stal, když už jsme věnovali takový usilí tomu ho nasetupovat. Nikde není řečený, že nemůžem v budoucnu udělat bug, že všechny joby co používají workspace v branchi skončí successem - to jestli ten job skončí successem je totiž rozhodnutí toho SUT.

Reply:
Doplneny check dat v DB